data/brands/amenity/fuel.json

```diff
@@ -2417,6 +2417,24 @@
         "name": "Hy-Vee Gas"
       }
     },
+    {
+      "displayName": "Iberdoex",
+      "locationSet": {"include": ["es"]},
+      "tags": {
+        "amenity": "fuel",
+        "brand": "Iberdoex",
+        "brand:wikidata": "Q25689041",
+        "brand:wikipedia": "ext:Iberdoex",
+        "name": "Iberdoex",
+        "post_office": "post_partner",
+        "post_office:parcel_from": "yes",
+        "post_office:parcel_pickup": "yes",
+        "post_office:service_provider" "GLS",
```

Comment:
Missing colon.
```suggestion
        "post_office:service_provider": "GLS",
```

Reply:
> Missing colon.

Thanks! I updated this (and missing colon in wikimedia_commons tag)